common/changes/@bentley/itwin-viewer-react/property-grid_2020-11-25-20-14.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@bentley/itwin-viewer-react",
+      "comment": "Switch to the property-grid-react package",
+      "type": "patch"
```

Comment:
should this actually be a minor bump?

Reply:
i thought about that, but it's not really adding a new feature. Functionally, there should minimal/no change